index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio Webpage</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li>
+          <a href="#welcome-section">Projects</a>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio</h1>
+    </section>
+
+    <a
+      rel="noopener"
+      id="profile-link"
+      href="https://github.com/omarbabou"
+      target="_blank"
+    ></a>
```

Comment:
Please use <a> tag inside an appropriate tag for the best practice of semantic HTML tags.